docs/samples/canary.yaml

```diff
@@ -7,8 +7,8 @@ spec:
     # 90% of traffic is sent to this model
     tensorflow:
       modelUri: "gs://mybucket/mymodel-2"
+  canaryTrafficPercent: 10
   canary:
```

Comment:
Should be expanded in the docs/samples/tensorflow README, i.e. usage of canary.yaml